app/assets/stylesheets/layout_components/siteheader.scss

```diff
@@ -95,15 +95,17 @@
     padding: 0 $base-padding;
   }
 
+  a{
+    margin: 0 auto;
```

Comment:
instead of 0 auto, can we do margin-left: auto; margin-right: auto;
I've noticed a few instances where this causes problems down the road